source/extensions/tracers/opentelemetry/opentelemetry_tracer_impl.cc

```diff
@@ -9,10 +9,13 @@
 #include "source/common/config/utility.h"
 #include "source/common/tracing/http_tracer_impl.h"
 
+#include "grpc_trace_exporter.h"
+#include "http_trace_exporter.h"
```

Comment:
Please use the full path

Reply:
Changed all in: [2a5d544](https://github.com/envoyproxy/envoy/pull/29207/commits/2a5d5448406e99fbfaecda651e0d176eda96383a)